soc/arm/nordic_nrf/nrf53/soc_cpu_idle.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file SoC extensions of cpu_idle.S for the Nordic Semiconductor nRF53 processors family.
+ */
+
+
+#if defined(_ASMLANGUAGE)
+
+.macro SOC_ON_EXIT_CPU_IDLE
+	nop
+	nop
+	nop
+	nop
+.endm
```

Comment:
This should be a `define` not a `.macro`.

Reply:
right, because you need:
```c
#define SOC_ON_EXIT_CPU_IDLE \
    nop; \
    nop; \
    nop; \
    nop;
```

Reply:
fixed